### PR TITLE
V3029 identical if expressions

### PR DESCRIPTION
**Pull request by gerhardol**
Static code analysis from gitextensions/gitextensions#5604
I do not believe these are critical, but they will at least simplify analysis in the solution

### CODE REVIEW

**Comment by drewnoakes** [comment]
This change is "incorrect". The static analysis tool does not take side effects into consideration.

The intent was to wait for the process to naturally exit, then check again to see whether it actually did before killing it.

With this change, it may attempt to kill and already exited program, resulting in an exception.

File: ConEmuWinForms/ConEmuSession.cs [hunk 1]
```diff
@@ -185,10 +185,11 @@ public void CloseConsoleEmulator()
 		{
 			try
 			{
-				if(!_process.HasExited)
-                    Thread.Sleep(10);
-                if (!_process.HasExited)
+				if (!_process.HasExited)
+				{
+					Thread.Sleep(10);
```
Comment (jbialobr) [inline ConEmuWinForms/ConEmuSession.cs:190]: Now it looks odd. Why should we wait 10msecs before killing the process. I think that the removed double checking was introduced on purpose, not by mistake.

Comment (jbialobr) [inline ConEmuWinForms/ConEmuSession.cs]: Now it looks odd. Why should we wait 10msecs before killing the process. I think that the removed double checking was introduced on purpose, not by mistake.

Comment (drewnoakes) [reply ConEmuWinForms/ConEmuSession.cs:190]: Agreed. I think this should be reverted.

GitHub is misbehaving at the moment. Site functions aren't working properly. Lots of double posting.

Comment (gerhardol) [reply ConEmuWinForms/ConEmuSession.cs:190]: I will revert and add a comment (the superproject PR is WIP due to this - as well as for your comments there).

**Pull request merged**